src/test/java/uk/ac/ebi/ega/egafuse/runner/CommandLineOptionParserTest.java

```diff
@@ -30,7 +30,6 @@
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
-import joptsimple.OptionException;
 import joptsimple.OptionSet;
 import uk.ac.ebi.ega.egafuse.model.CliConfigurationValues;
 
```

Comment:
If you use $@ or $* in the fuseclient.sh, then the user doesn't have to put the username and password into quotation marks (which is a bit weird): `./fuseclient.sh start "--u=USERNAME --p=PASSWORD"`
https://unix.stackexchange.com/questions/129072/whats-the-difference-between-and